rpc/Cargo.toml

```diff
@@ -9,6 +9,9 @@ thiserror = "1.0.40"
 jsonrpsee = { version = "0.20", features = ["client-core", "macros"] }
 celestia-types = { workspace = true }
 serde = { version = "1.0.188", features = ["derive"] }
+libp2p = "0.52.3"
+libp2p-identity = "0.2.3"
```

Comment:
I'd enable it as a libp2p feature instead to make sure the versions are intact

Reply:
yes, looks like I didn't need to import `-identity` directly, but could use it from `libp2p`